.release-it.cjs

```diff
@@ -0,0 +1,38 @@
+"use strict";
```

Comment:
Yes, I noted in the PR that I planned on updating the release docs after this PR is merged. I figured I'd update the wiki, but it could be a README in the `build/release`?

Reply:
A README in `build/release` works for me.